docs/index.html.ejs

```diff
@@ -635,6 +635,48 @@
       </div>
     </section>
 
+    <section class="component">
+      <h4 id="status-bar">Status Bar</h4>
```

Comment:
Wanna link to this in the TreeView on the left side as well?

Reply:
Oh I missed that, just fixed it.